sci-libs/cantera/cantera-2.4.0.ebuild

```diff
@@ -0,0 +1,130 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6} )
+
+inherit desktop python-single-r1 scons-utils toolchain-funcs
+
+DESCRIPTION="Object-oriented tool suite for chemical kinetics, thermodynamics, and transport"
+HOMEPAGE="http://www.cantera.org"
+SRC_URI="https://github.com/Cantera/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+cti cxx fortran pch +python static-libs test"
+
+REQUIRED_USE="
+	|| ( cxx python )
+	cti? ( ${PYTHON_REQUIRED_USE} )
+	fortran? ( cxx )
+	python? ( cti )
+	static-libs? ( cxx )
+	^^ ( $(python_gen_useflags 'python3*') )
+	"
+
+RDEPEND="
+	python? (
+		dev-python/numpy[${PYTHON_USEDEP}]
+	)
+"
+
+DEPEND="
+	${RDEPEND}
+	dev-cpp/eigen
+	dev-libs/boost
+	dev-libs/libfmt:0=
+	dev-util/scons[${PYTHON_USEDEP}]
+	sci-libs/sundials
```

Comment:
Is this unnecessary at runtime? Also `:=` seems correct, given it's subslotted.

Reply:
I will recheck requirement of `:=` for fmt (e.g it's required by runtime). Sundials and Eigen are really required by C++ cantera library at runtime but they are both seems aren't required by runtime of python module. Therefore currently I didn't placed it to RDEPEND. At least `:=`  actually is required to don't care if `cxx` USE-flag is chosen by user or not.

Reply:
Done: 
`sci-libs/sundials` moved to RDEPEND with addition of `:0=` SLOT dependency.
Also `:0=` is removed for `dev-libs/libfmt` dependecy.